ac_dc/deduplicate.py

```diff
@@ -412,9 +414,9 @@ def remove_duplicates(
 
     def process(record):
         for dup in record["duplicates"]:
-            if str(record["id"]) == dup:
+            if int(record["id"]) == dup or dup == -1:
```

Comment:
What does dup == -1 look for? Is this a boundry test?

Reply:
I put `[-1]` as a default value in `find_deduplicates` to follow a consistent schema `List[int]`.